### PR TITLE
Add NBD INFINITY 200RS config

### PR DESCRIPTION
**Pull request by yangyeah**
Add NewBeeDrone Infinity 200RS target.

### CODE REVIEW

File: configs/default/NEBD-NBD_INFINITY200RS.config [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+set gyro_hardware_lpf = OPTION_2
+set gyro_lpf1_static_hz = 0
+set gyro_lpf2_type = PT2
+set gyro_lpf2_static_hz = 145
+set dyn_notch_count = 1
+set dyn_notch_q = 500
+set dyn_notch_min_hz = 150
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set acc_lpf_hz = 10
```
Comment (haslinghuis) [inline configs/default/NEBD-NBD_INFINITY200RS.config:102]: Please remove user / custom / tune settings (we have presets for that)
```suggestion
```

File: configs/default/NEBD-NBD_INFINITY200RS.config [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+set osd_sys_delay_pos = 341
+set osd_sys_distance_pos = 341
+set osd_sys_lq_pos = 341
+set osd_sys_goggle_dvr_pos = 341
+set osd_sys_vtx_dvr_pos = 341
+set osd_sys_warnings_pos = 341
+set osd_sys_vtx_temp_pos = 341
+set osd_sys_fan_speed_pos = 341
+set osd_canvas_width = 30
+set osd_canvas_height = 13
```
Comment (haslinghuis) [inline configs/default/NEBD-NBD_INFINITY200RS.config:192]: Same for OSD settings.
```suggestion
```

File: configs/default/NEBD-NBD_INFINITY200RS.config [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+set max7456_spi_bus = 3
+set flash_spi_bus = 1
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 4
+set gyro_1_sensor_align = CW90
+set gyro_1_align_yaw = 900
+set gyro_2_spibus = 2
+set craft_name = INFINITY 200RS
```
Comment (haslinghuis) [inline configs/default/NEBD-NBD_INFINITY200RS.config:200]: Same.
```suggestion
```

**Comment by yangyeah** [comment]
I have pushed a new commint for remove the custom settings, please cheak it, thank you.

**Comment by nerdCopter** [comment]
i could be wrong, but i believe these are pilot options and/or set by betaflight defaults and should be removed.  @haslinghuis , can you confirm or refute?
```
set serialrx_provider = SBUS
set blackbox_device = SPIFLASH
set dshot_burst = ON
set dshot_bidir = ON
set motor_pwm_protocol = DSHOT600
```

**Comment by haslinghuis** [comment]
| setting | comment |
| :--- | :--- |
| set serialrx_provider = SBUS | remove |
| set blackbox_device = SPIFLASH | necessary | 
| set dshot_burst = ON |  remain |
| set dshot_bidir = ON | depends on ESC firmware |
| set motor_pwm_protocol = DSHOT600 | remove |

**Comment by yangyeah** [comment]
I have push the settings, please check it, thank you.

|  setting | comment |
|  ----  | ----  |
| set serialrx_provider = SBUS | remove |
| set dshot_bidir = ON | remove |
| set motor_pwm_protocol = DSHOT600 | remove |

**Comment by haslinghuis** [comment]
Thanks, now we only need to add defines for hardware:

http://betaflight.com/docs/manufacturer/manufacturer-design-guidelines#42-definitions-for-targets

**Comment by yangyeah** [comment]
Should I add the defines on the top of config file?

#define USE_ACCGYRO_BMI270
#define USE_GYRO_SPI_MPU6000
#define USE_ACC_SPI_MPU6000
#define USE_FLASH_M25P16
#define USE_FLASH_W25M
#define USE_FLASH_W25N01G
#define USE_MAX7456

**Comment by haslinghuis** [comment]
You can add these on line 5:

    #define USE_GYRO_SPI_MPU6000
    #define USE_ACC_SPI_MPU6000
    #define USE_ACCGYRO_BMI270
    #define USE_FLASH_W25N01G
    #define USE_MAX7456

**Comment by yangyeah** [comment]
I have added it, should I add the defines for other nebd configs?

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
Yes please. This will enhance user experience (if it works out-of-the-box)

**Comment by yangyeah** [comment]
Hi, what sholud I do for that, it can be merged?

**Review by nerdCopter — APPROVED** [review]
* approving, but please verify before merging. (i skim-read only)

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**